runtime/src/launch/mpirun/launch-mpirun.c

```diff
@@ -38,7 +38,7 @@ static char** chpl_launch_create_argv(const char *launch_cmd,
   int numranks;
 
   // TODO When MPI Comm layer is supported, support -nl>1 && CHPL_COMM!=none
-  if (numLocales != 1 || mpi_num_ranks==NULL) {
+  if (numLocales != 1) {
     chpl_error("For mpirun, specify number of SPMD images via --spmd <#>", 0, 0);
   }
```

Comment:
This makes the launcher stop with the given message and an error, if nl > 1... is it clear enough to the user that mpirun's problem is nl > 1 ?